src/components/sunburst/sunburst-container.jsx

```diff
@@ -105,51 +112,70 @@ export default class SunburstContainer extends React.Component {
   }
 
   _fetchTimestampSimhashData () {
-    const url = `${this.props.conf.waybackDiscoverDiff}/simhash?url=${encodeURIComponent(this.props.url)}&timestamp=${this.state.timestamp}`;
+    const url = this.props.conf.waybackDiscoverDiff + '/simhash?url=' + encodeURIComponent(this.props.url) +
+      '&timestamp='+ this.state.timestamp;
     fetch_with_timeout(fetch(url)).then(response => {return checkResponse(response);})
       .then(response => response.json())
       .then((jsonResponse) => {
         if (jsonResponse['status']) {
           throw Error(jsonResponse['message']);
         }
-        const json = this._decodeJson(jsonResponse);
+        const json = this._decodeCompressedJson(jsonResponse);
         this._fetchSimhashData(json);
       })
       .catch(error => {this.errorHandled(error.message);});
   }
 
   _fetchSimhashData (timestampJson) {
-    const url = `${this.props.conf.waybackDiscoverDiff}/simhash?url=${encodeURIComponent(this.props.url)}&year=${this.state.timestamp.substring(0, 4)}`;
+    let url = this.props.conf.waybackDiscoverDiff + '/simhash?url=' + encodeURIComponent(this.props.url) + '&year=' +
+      this.state.timestamp.substring(0, 4);
+    if (this.props.conf.compressedSimhash) url +='&compress=1';
```

Comment:
For readability reasons, please don't make `if` in one line. Please put `url +='&compress=1'` in the next line.